### ISSUE TITLE
ASU Logo and Home Page Icon are Missing Title Tags

### ISSUE THREAD

**Issue by wmcconnell** [ASU-KE/UDS-WordPress-Theme#305]
The theme was missing two `title` tags that need to be in place for compliance with the [Web Standards Checklist](https://brandguide.asu.edu/execution-guidelines/web/standards-checklist):

- The main ASU logo should have a `title` tag that says 'ASU home'
- The home icon on the main menu bar should have a `title` tag with the site's title followed by the word 'home' .

These are for the browser to use when displaying 'tool tips' over links. For example, hovering your mouse over the home icon on the Biodesign site should result in a tip that says 'Biodesign home'. These **are not** our UDS Tool Tips, but a browser feature.

**Examples**:

<img width="829" alt="Screen Shot 2022-01-14 at 2 40 24 PM" src="https://user-images.githubusercontent.com/31013359/149589039-e265817d-e749-4161-ad45-06078285d6c2.png">

<img width="756" alt="Screen Shot 2022-01-14 at 2 38 50 PM" src="https://user-images.githubusercontent.com/31013359/149589145-9843c92f-b285-4847-abac-cd772e4fa062.png">
